src/topFrame.cpp

```diff
@@ -32,18 +32,179 @@ extern int g_playFileFromRadioEventId;
 extern int g_recFileFromModulatorEventId;
 extern int g_txLevel;
 
-// Override for wxAuiNotebook to prevent tabbing to it.
-class TabFreeAuiNotebook : public wxAuiNotebook
+// THIS IS VERY MUCH A HACK! wxTabFrame is not in the public interface and should
+// not be here. Unfortunately this is needed to get the tab state loaded and
+// saved. Here's hoping this interface remains stable.
+//
+// (Last retrieved from wxWidgets 3.2 on August 8, 2023.)
+class wxTabFrame : public wxWindow
```

Comment:
This is the change I'm not comfortable with considering it's a non-public interface from wxWidgets itself. I don't *think* it changes all that often but still.